pollen.json

```diff
@@ -69,5 +69,9 @@
 	"DownloadDirectory": null,
 	"DownloadBubbleEnabled": null,
 	"OsColorMode": null,
-	"ArcPolicy": null
+	"ArcPolicy": null,
+	"InstantTetheringAllowed": true,
+	"SmartLockSigninAllowed": true,
+	"NearbyShareAllowed": true,
+	"WallpaperImage": {}

```

Comment:
maybe not this one since some districts don't have their school wallpaper posted elsewhere on the internet

Reply:
if someone wants to be as inconspicuous as possible

Reply:
yeah, was just gonna write that too there's a reason i hadn't added it

others look fine